Cannot compile registration

```
/home/victor/pcl/registration/src/ppf_registration.cpp:45:62: error: expected constructor, destructor, or type conversion before ‘PCL_INSTANTIATE_PPFRegistration’
 PCL_INSTANTIATE_PRODUCT(PPFRegistration, (PCL_XYZ_POINT_TYPES)(PCL_NORMAL_POINT_TYPES));
                                                              ^
make[2]: *** [registration/CMakeFiles/pcl_registration.dir/src/ppf_registration.cpp.o] Erreur 1
```

http://www.pcl-users.org/Error-compiling-pcl-registration-in-PCL-1-7-2-td4035504.html

--- a/registration/include/pcl/registration/ppf_registration.h
+++ b/registration/include/pcl/registration/ppf_registration.h
@@ -285,7 +285,7 @@ namespace pcl
                               Eigen::Affine3f &pose2);
   };
 }
-#ifdef PCL_NO_PRECOMPILE
+
 #include <pcl/registration/impl/ppf_registration.hpp>
-#endif // PCL_NO_PRECOMPILE
+
 #endif // PCL_PPF_REGISTRATION_H_

--- a/registration/src/ppf_registration.cpp
+++ b/registration/src/ppf_registration.cpp
@@ -35,15 +35,16 @@
  * $Id$
  *
  */
-#include <pcl/registration/ppf_registration.h>
- 
-#ifndef PCL_NO_PRECOMPILE
-#include <pcl/point_types.h>
-#include <pcl/impl/instantiate.hpp>
-#include <pcl/registration/impl/ppf_registration.hpp>
 
-PCL_INSTANTIATE_PRODUCT(PPFRegistration, (PCL_XYZ_POINT_TYPES)(PCL_NORMAL_POINT_TYPES));
-#endif    // PCL_NO_PRECOMPILE
+#include <pcl/registration/ppf_registration.h>
+
+//#ifndef PCL_NO_PRECOMPILE
+//#include <pcl/point_types.h>
+//#include <pcl/impl/instantiate.hpp>
+//#include <pcl/registration/impl/ppf_registration.hpp>
+
+//PCL_INSTANTIATE_PRODUCT(PPFRegistration, (PCL_XYZ_POINT_TYPES)(PCL_NORMAL_POINT_TYPES));
+//#endif    // PCL_NO_PRECOMPILE
 
 void
 pcl::PPFHashMapSearch::setInputFeatureCloud (PointCloud<PPFSignature>::ConstPtr feature_cloud)

